docs/src/guide/bastion.md

```diff
@@ -0,0 +1,48 @@
+## Bastion
+
+The bastion instance provides a public SSH entrypoint into the VCN from which resources in private subnets may be accessed - recommended to limit public IP usage and exposure.
+
+### General
+
+The bastion host parameters concern:
+0. whether you want to enable the bastion
+0. from where you can access the bastion
+0. the different parameters about the bastion host e.g. shape, image id etc.
+
+### Cloud Init
+
+Custom startup configuration for the created bastion instance.
+
+### Image
+
+The OS image for the created bastion instance.
+
+See also:
+* [Oracle Autonomous Linux 7.x](https://docs.oracle.com/en-us/iaas/images/autonomous-linux-7x)
+* [Oracle Autonomous Linux 8.x](https://docs.oracle.com/en-us/iaas/images/autonomous-linux-8x)
+
+### SSH
+
+Command usage for `ssh` through the created bastion to the operator host is included in the module's output:
+```
+$ terraform output
```

Comment:
Maybe we can also generate an output that the user can append to the ~/.ssh/config?